Source/GitReleaseManager.IntegrationTests/ReleaseNotesBuilderIntegrationTests.cs

```diff
@@ -19,6 +21,8 @@ namespace GitReleaseManager.IntegrationTests
     [TestFixture]
     public class ReleaseNotesBuilderIntegrationTests
```

Comment:
Can you perhaps try adding `[Explicit]` to the class?
I am hoping this will prevent the Configure method to run (although I have no facts to back that up).